garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stools/faceposer.lua

```diff
@@ -335,10 +335,16 @@ function TOOL.BuildCPanel( CPanel, faceEntity )
 
 	CPanel:AddItem( QuickFace )
 
-	CPanel:AddControl( "Slider", { Label = "#tool.faceposer.scale", Command = "faceposer_scale", Type = "Float", Min = -5, Max = 5, Help = true, Default = 1 } ):SetHeight( 16 )
-	CPanel:AddControl( "Button", { Text = "#tool.faceposer.randomize", Command = "faceposer_randomize" } )
+	-- Scale
+	local size = CPanel:NumSlider( "#tool.faceposer.scale", "faceposer_scale", -5, 5, 2 )
+	CPanel:ControlHelp( "#tool.faceposer.scale.help" )
+	size:SetDefaultValue( 1 )
+	size:SetHeight( 16 )
```

Comment:
Pointless comment + coding practice + pointless SetDefaultValue call again.

If you declare a variable (`size`), use it immediately.
```
local size = CPanel:NumSlider( "#tool.faceposer.scale", "faceposer_scale", -5, 5, 2 )
CPanel:ControlHelp( "#tool.faceposer.scale.help" )
size:SetDefaultValue( 1 )
size:SetHeight( 16 )
```
Should be
```
local size = CPanel:NumSlider( "#tool.faceposer.scale", "faceposer_scale", -5, 5, 2 )
size:SetDefaultValue( 1 )
size:SetHeight( 16 )
CPanel:ControlHelp( "#tool.faceposer.scale.help" )
```
But really it could be 
```
CPanel:NumSlider( "#tool.faceposer.scale", "faceposer_scale", -5, 5, 2 ):SetHeight( 16 )
CPanel:ControlHelp( "#tool.faceposer.scale.help" )
```